pkg/apis/extensions/types.go

```diff
@@ -42,44 +42,6 @@ const (
 	SysctlsPodSecurityPolicyAnnotationKey string = "security.alpha.kubernetes.io/sysctls"
 )
 
-// describes the attributes of a scale subresource
```

Comment:
Just to verify - removing these will not break backward compatibility, right?

Reply:
Correct, as seen by there being zero changes in the openapi/swagger schemas (only apps/v1 additions), or external API types, or client-go methods.